.gitignore

```diff
@@ -101,3 +101,6 @@ deps/
 android-toolchain-arm/
 cc-toolchain/
 /TraceOutput.txt
+Build/VcBuild
```

Comment:
These already in .gitignore? https://github.com/Microsoft/ChakraCore/blob/master/.gitignore#L42